app/views/donors/show.html.erb

```diff
@@ -1,9 +1,37 @@
-<div id="edit_partial">
+<div id="edit_partial" class="custom-donor-details">
   <%= link_to edit_donor_url, remote: true do %>
   <i class="fa fa-pencil fa-lg"></i>
   <% end %>
 </div>
 
-<div id="show_partial">
+<div id="show_partial" class="custom-donor-details custom-bottom-padding">
   <%= render "show" %>
 </div>
+
+<div class="custom-total-donated">
+    <h2>Total Amount Donated</h2>
```

Comment:
Please use two spaces (rather than four) for indentation. I think your editor might be set to four for ERB files. 😀